sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/util/AzureSpringIdentifier.java

```diff
@@ -41,6 +41,8 @@ private AzureSpringIdentifier() {
     public static final String VERSION = getVersion();
     public static final String AZURE_SPRING_APP_CONFIG = "az-sp-cfg/" + VERSION;
     public static final String AZURE_SPRING_EVENT_HUBS = "az-sp-eh/" + VERSION;
+    //Kafka supports set client software name and version individually.
+    public static final String AZURE_SPRING_EVENT_HUBS_KAFKA_OAUTH = "az-sp-kafka/";
```

Comment:
maybe we can remove the trailing `/`

Reply:
I think we can reserver it in case other things appended